articles/downloading_data.md

```diff
@@ -102,6 +102,29 @@ entity <- synGet("syn3260973", version=1)
 
 See [versioning](http://docs.synapse.org/articles/versioning.html) for more details.
 
+### Links
+
+There are multiple ways of downloading a link in Synapse.  Without specifying the followLink parameter, you will only retrieve the link itself without downloading wherever the link leads to.  There is currently not a way to follow the link in R.
```

Comment:
what are the multiple ways?

Why is there not a way to follow the link in R? We have `synapser` now.